mlflow/cli.py

```diff
@@ -242,7 +242,7 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
 @click.option("--waitress-opts", default=None,
               help="Additional command line options for waitress-serve.")
 @click.option("--expose-prometheus", default=None,
-              help="Path to the directory where metrics will be stored. If the directory"
+              help="Path to the directory where metrics will be stored. If the directory "
                    "doesn't exist, it will be created."
```

Comment:
It looks like there is also missing white space after the sentence ending on line 246. Can you add it as well?

Reply:
@tomasatdatabricks good catch.
Thanks for the review. updated it.